src/components/CreateVmWizard/steps/BasicSettings.js

```diff
@@ -116,6 +116,7 @@ class BasicSettings extends React.Component {
     this.handleChange = this.handleChange.bind(this)
     this.validateForm = this.validateForm.bind(this)
     this.validateVmName = this.validateVmName.bind(this)
+    this.defaultValues = props.data
```

Comment:
This should be an explicitly passed prop, not a copy of the initial values.

But if we change to use values from the default template instead of the hard coded values, this won't be needed at all.

Reply:
I changed my code according your review